module/Motion/Motion.cpp

```diff
@@ -0,0 +1,9 @@
+/**
+ * @file   Motion.cpp
+ * @brief  動作の親クラス
+ * @author kodama0720 mutotaka0426
```

Comment:
```suggestion
 * @author desty505
```
ここは今年のリポジトリに追加した人で良いよ